13-Weather-App/config.js

```diff
@@ -0,0 +1,3 @@
+var config = {
```

Comment:
Please don't share your APIs keys publicly on the internet, it can be mis-used.

Reply:
That's why I added gitignore file to hide the API keys in config.js but you requested to remove it.